hll_seed_vip/utils.py

```diff
@@ -24,8 +24,26 @@
     ServerPopulation,
     VipPlayer,
 )
+from hll_seed_vip.constants import INDEFINITE_VIP_DATE
 
 
+def has_indefinite_vip(player: VipPlayer) -> bool:
```

Comment:
Could you change the typing to fix the pyright complaint, `player: VipPlayer | None`

Reply:
Done.